packages/contents/src/drivefs.ts

```diff
@@ -0,0 +1,517 @@
+export const DIR_MODE = 16895; // 040777
```

Comment:
a lot of this looks copied: probably needs some headers with URLs and license info from the sources.